crates/revm/src/instructions/arithmetic.rs

```diff
@@ -1,15 +1,16 @@
 use crate::{gas, Interpreter, Return, Spec};
 
 use super::i256::{i256_div, i256_mod};
-use core::{convert::TryInto, ops::Rem};
-use primitive_types::{U256, U512};
+use core::ops::Rem;
+use ruint::aliases::U256;
+use std::ops::{Div, Mul};
 
 pub fn div(op1: U256, op2: U256) -> U256 {
-    if op2.is_zero() {
-        U256::zero()
+    if op2 == U256::ZERO {
+        U256::ZERO
     } else {
         //op1 / op2
-        super::i256::div_u256::div_mod(op1, op2).0
+        op1.div(op2)
     }
```

Comment:
Alternatively instead of the `if` you can do `op1.checked_div(op2).unwrap_or_default()`.